latch_cli/snakemake/config/utils.py

```diff
@@ -115,8 +118,23 @@ def parse_type(
             for x in v
         )
         if len(set(parsed_types)) > 1:
-            raise ValueError("all types in List must be same")
-        return List[parsed_types[0]]
+            raise ValueError(
+                "Generic Lists are not supported - please ensure that all elements in a"
+                " list are of the same type",
+            )
+        typ = parsed_types[0]
+        if typ in {LatchFile, LatchDir}:
+            click.secho(
+                dedent(f"""
+                Failed to parse {name}. Lists containing LatchFile or LatchDir
+                are not supported.
+                """),
+                fg="red",
+            )
+            raise click.exceptions.Exit(1)
+            # TODO: (rahul) add back once console PR is merged
```

Comment:
```suggestion
            # todo(rahul): add back once console PR is merged
```

minor stylistic thing we do